osu.Game/Screens/Edit/Setup/ResourcesSection.cs

```diff
@@ -70,7 +73,7 @@ private void load()
                 audioTrackTextBox = new FileChooserLabelledTextBox
                 {
                     Label = "Audio Track",
-                    Current = { Value = Beatmap.Value.Metadata.AudioFile ?? "Click to select a track" },
+                    Current = { Value = Beatmap.Metadata.AudioFile ?? "Click to select a track" },
```

Comment:
Slightly confused by this; it's the only use of `Beatmap` and not `working` in the entire file. Is this an oversight, or intended behaviour?

Reply:
I'll change this for consistency. Metadata is one of the pieces that isn't clone, so both of these are actually pointing at the same instance.